SampleUsages/Program.cs

```diff
@@ -42,30 +77,48 @@ public void RunScenario(string scenarioFilePath)
             Console.WriteLine("Start running scenarios.");
             var counter = 0;
             var now = DateTime.UtcNow;
+            var repo = new TestRepository();
+
+            var scenario = repo.AddTestScenario(new ServerlessResultManager.TestScenario
+            {
+                Name = string.Format("Scenario-{0}-{1}", scenarioFilePath, now.ToString("s")),
+                StartTimeUtc = now
+            });
 
             foreach (var testScenario in testScenarios)
             {
                 Console.WriteLine($"Start running scenario for function {testScenario.FunctionName} {++counter}/{testScenarios.Count}.");
                 var testFolder = string.Format(testScenario.FunctionName);
                 Directory.CreateDirectory(testFolder);
```

Comment:
that is not quite right. Due to the architecture of the system, you can't tell if AzurePerformanceResultsProvider is not called from the outside. Some different path then RunScenario. Especially when you run single function tests. That fixes the issue. Generally what's a good practice is to make sure every time you write, that directory exists. In our case, if it's not we want to create it anyway. Directory.Create(path) will do nothing if the directory is already created.